### ISSUE TITLE
Pillow requires Python '>=3.7' but the running Python is 3.6.9

### ISSUE THREAD

**Issue by italianconcerto** [fcakyon/yolov5-pip#113]
I'm trying to run yolov5 on a docker inside a jetson nano with JetPack 4.6.1. 
I'm using Python 3.6.9 but when I run "pip3 install yolov5" I get the error in the title.
Is there a way to go on with that?
Thanks

**Comment by fcakyon** [comment]
@italianconcerto you have to clone the repo, change the requirement versions and install the package from source to make it work with jetson nano.

You can follow this discussion: https://github.com/ultralytics/yolov5/issues/2524#issuecomment-1120462526

**Comment by italianconcerto** [comment]
I tried but I can't find a way to make torch, torchvision and opencv work together with CUDA. the only way seems to be using this: https://catalog.ngc.nvidia.com/orgs/nvidia/containers/l4t-ml

This contain the dockerized libraries already working on jetson nano. 
Unfortunately that doesn't seem to be working with the last version of YOLOv5 since I tried cloning it and running detect.py but it didn't work.

Manually compiling everything is a huge waste of time since this docker image works, is there a way to make YOLOv5 work on that container?

Thanks
